src/Components/HomePage/FeaturedBannersSlider.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import APIcall from './APIcall';
+import { Slider } from '../Slider/Slider';
+
+const Container = styled.section`
+    width: 100%;
+    height: 90vh;
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row;
+    overflow: hidden;
+    margin-bottom: 3rem;
+`
+
+const FeaturedBannersSlider = () => {
+    const [FetchedData, setFetchedData] = React.useState(null);
```

Comment:
Since this is not a component nor a class, avoid PascalCase. Instead this should be named as `fetchedData` in camelCase.